src/meshlink++.h

```diff
@@ -294,7 +300,7 @@ class mesh {
 		meshlink_set_node_duplicate_cb(handle, &node_duplicate_trampoline);
 		meshlink_set_log_cb(handle, MESHLINK_DEBUG, &log_trampoline);
 		meshlink_set_channel_accept_cb(handle, &channel_accept_trampoline);
-		meshlink_set_connection_try_cb(handle, &connection_try_trampoline);
+		//meshlink_set_connection_try_cb(handle, &connection_try_trampoline);
```

Comment:
Why is this line commented out?

Reply:
When I uncommented it and try to build library it couldn't link `meshlink_set_connection_try_cb` function in `meshlink.c`. I actually wrote about this and `chatpp.cc` build issue in an email but unfortunately it was lost before reaching.